common/platform.cpp

```diff
@@ -20,6 +20,7 @@
  */
 
 #include "common/platform.h"
+#include "common/gui_options.h"
```

Comment:
I do not like this dependency. Please, move the new methods to gui_options.cpp, not here.

`common/patform.h` is one of the basic includes and is used everywhere. Making gui_options.h dependency is an overkill

Reply:
Here it's platform.cpp so this is not so critical.